### PR TITLE
Event manager #2

### PR DESCRIPTION
**Pull request by wozane**

### CODE REVIEW

File: event_manager/lib/validate_phone_number.rb [hunk 1]
```diff
@@ -1,12 +1,18 @@
 class ValidatePhoneNumber
   def call(number)
-    return "Invalid phone format" unless number.kind_of?(String)
+    return 'Invalid phone format' unless number.is_a?(String)
```
Comment (esse) [inline event_manager/lib/validate_phone_number.rb]: I believe in this approach it we would be better to raise an exception instead

File: event_manager/lib/validate_phone_number.rb [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
 
     case number.length
     when 10
       number
+    when 11
+      if number[0] == '1'
+        number[1..-1]
+      else
+        'Invalid phone number'
```
Comment (esse) [inline event_manager/lib/validate_phone_number.rb]: Same here - exception would be better

File: event_manager/lib/validate_phone_number.rb [hunk 3]
```diff
@@ -1,12 +1,18 @@
 class ValidatePhoneNumber
   def call(number)
```
Comment (esse) [inline event_manager/lib/validate_phone_number.rb]: I would rather use method name `validate` and class method, i.e. `self.validate` name here.

**Review by esse — CHANGES_REQUESTED** [review]
Some changes required, however - overall: good code 👍

**Comment by wozane** [comment]
request for review before merge @esse

**Review by esse — APPROVED** [review]
👍

**Comment by wozane** [comment]
Branch merged manually

**Pull request closed without merging**